### PR TITLE
Add Microsoft.AAD/DomainServices Swagger Spec

### PR DESCRIPTION
**Pull request by jihochang**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by jihochang** [comment]
Our service is now public, so moving the swagger from azure-rest-api-specs-pr to azure-rest-api-specs branch.

Here is the link to the previous pull request to azure-rest-api-specs-pr branch:
https://github.com/Azure/azure-rest-api-specs-pr/pull/67

No changes have been made, since last pull request. I was unable to add the previous reviewer veronicagg to this pull request.

**Comment by salameer** [comment]
Adding Veronica, Since she has the context

**Comment by veronicagg** [comment]
@jihochang our repo structure changed between your private PR and this one. 
Could you please add a "readme.md" file to match the new one? similar to https://github.com/Azure/azure-rest-api-specs/blob/current/specification/advisor/resource-manager/readme.md You can find more  readme.md examples at https://github.com/Azure/autorest/tree/97b68250afd96111f79047e24e22eeb82a30426f/src/autorest-core/test/variations

**Review by veronicagg — CHANGES_REQUESTED** [review]
Please see my comment regarding adding a readme.me file.

**Comment by jihochang** [comment]
I have added the readme file and changed the file structure to match the new file structure.

File: specification/domainservices/resource-manager/Microsoft.AAD/2017-01-01/DomainServicesSwagger.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
         },
         "x-ms-examples": {
           "Get Operations": {
-            "$ref": "../examples/GetOperations.json"
+            "$ref": "/examples/GetOperations.json"
```
Comment (veronicagg) [inline specification/domainservices/resource-manager/Microsoft.AAD/2017-01-01/DomainServicesSwagger.json]: I believe this references have to be "./examples..." so start with "." - https://travis-ci.org/Azure/azure-rest-api-specs/jobs/277044016

**Review by veronicagg — CHANGES_REQUESTED** [review]

**Comment by jihochang** [comment]
Updated Example Links

**Review by veronicagg — APPROVED** [review]
LGTM, thanks!

**Pull request merged**